xbmc/pvr/channels/PVRChannelGroup.h

```diff
@@ -432,6 +434,11 @@ namespace PVR
      */
     bool IsMissingChannelsFromClient(int iClientId) const;
 
+    /*!
+     * @brief For each channel and it's corresponding epg channel data update the order from the group members
```

Comment:
I'm not a native English speaker, but is "it's" correct here? I thought it has to read "its", no?

Reply:
Yes, you are correct.